fhttp/http.go

```diff
@@ -1151,13 +1156,23 @@ func EchoHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
-var payload = make([]byte, MaxPayloadSize)
-
 func init() {
+	if len(payload) == 0 {
+		ChangeMaxPayloadSize(MaxPayloadSizeKb)
+	}
+}
+
+// ChangeMaxPayloadSize is used to change max payload size and fill it with pseudorandom content
+func ChangeMaxPayloadSize(newMaxPayloadSizeKb int) {
+	MaxPayloadSizeKb = newMaxPayloadSizeKb
```

Comment:
we can either have 2 variables or just the bytes one or use len(payload) for the bytes value